### PR TITLE
Changelog for v46.13.0

### PR DESCRIPTION
**Pull request by brpowell**
### What does this PR do?

Changelog for v46.13.0

### CODE REVIEW

**Comment by brpowell** [comment]
@hbc-bindu change log for 46.13.0 is ready for review.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+
+- Prevent 'SFDX: Diff File Against Org' command from appearing when a non-sfdx project is opened ([PR #1608](https://github.com/forcedotcom/salesforcedx-vscode/pull/1608), [Issue #1600](https://github.com/forcedotcom/salesforcedx-vscode/issues/1600))
+
+- Update documentation link when guiding user to fix Java configuration ([PR #1602](https://github.com/forcedotcom/salesforcedx-vscode/pull/1602))
+
+## Added
+
+#### docs
+
+- Add documentation for setting up Windows PATH variable to use the Salesforce CLI ([PR #1621](https://github.com/forcedotcom/salesforcedx-vscode/pull/1621))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md]: Can we add the name of the new section and link to it ?

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,27 @@
+# 46.13.0 - September 5, 2019
+
+## Fixed
+
+#### salesforcedx-vscode-apex
+
+- Fix syntax highlighting issue with String when clauses in Apex switch statements ([PR #1628](https://github.com/forcedotcom/salesforcedx-vscode/pull/1628), [Issue #967](https://github.com/forcedotcom/salesforcedx-vscode/issues/967))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md]: Based on the docs we might want to change this to `... String literals for when values in Apex Switch Statements` (https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/langCon_apex_switch.htm)

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,27 @@
...
+
+- Fix syntax highlighting issue with block comments without spaces ([PR #1628](https://github.com/forcedotcom/salesforcedx-vscode/pull/1628), [Issue #921](https://github.com/forcedotcom/salesforcedx-vscode/issues/921))
+
+- Fix syntax highlighting issue with Apex namespace names in Apex class names ([PR #1628](https://github.com/forcedotcom/salesforcedx-vscode/pull/1628))
+
+#### salesforcedx-vscode-core
+
+- Prevent 'SFDX: Diff File Against Org' command from appearing when a non-sfdx project is opened ([PR #1608](https://github.com/forcedotcom/salesforcedx-vscode/pull/1608), [Issue #1600](https://github.com/forcedotcom/salesforcedx-vscode/issues/1600))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md]: The command name needs to use the apostrophe character instead of single quotes so it gets styled differently.

Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md]:
```suggestion
- Prevent `SFDX: Diff File Against Org` command from appearing when a non-SFDX project is opened ([PR #1608](https://github.com/forcedotcom/salesforcedx-vscode/pull/1608), [Issue #1600](https://github.com/forcedotcom/salesforcedx-vscode/issues/1600))
```

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 2 above)
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md]:
```suggestion
- Fix syntax highlighting issue with String literals for `when` values in Apex Switch Statements ([PR #1628](https://github.com/forcedotcom/salesforcedx-vscode/pull/1628), [Issue #967](https://github.com/forcedotcom/salesforcedx-vscode/issues/967))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,27 @@
...
+
+- Prevent `SFDX: Diff File Against Org` command from appearing when a non-SFDX project is opened ([PR #1608](https://github.com/forcedotcom/salesforcedx-vscode/pull/1608), [Issue #1600](https://github.com/forcedotcom/salesforcedx-vscode/issues/1600))
+
+- Update documentation link when guiding user to fix Java configuration ([PR #1602](https://github.com/forcedotcom/salesforcedx-vscode/pull/1602))
+
+## Added
+
+#### docs
+
+- Add documentation for setting up Windows PATH variable to use the Salesforce CLI ([PR #1621](https://github.com/forcedotcom/salesforcedx-vscode/pull/1621))
```
Comment (brpowell) [inline packages/salesforcedx-vscode/CHANGELOG.md]:
```suggestion
- Add documentation for setting up Windows PATH variable under [Set Salesforce CLI Path (Windows)](https://forcedotcom.github.io/salesforcedx-vscode/articles/troubleshooting#set-salesforce-cli-path-windows)([PR #1621](https://github.com/forcedotcom/salesforcedx-vscode/pull/1621))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,27 @@
...
+
+- Fix syntax highlighting for literal `when` values of String type in Apex switch statement ([PR #1628](https://github.com/forcedotcom/salesforcedx-vscode/pull/1628), [Issue #967](https://github.com/forcedotcom/salesforcedx-vscode/issues/967))
+
+- Fix syntax highlighting for block comments without spaces ([PR #1628](https://github.com/forcedotcom/salesforcedx-vscode/pull/1628), [Issue #921](https://github.com/forcedotcom/salesforcedx-vscode/issues/921))
+
+- Fix syntax highlighting for namespace names in Apex classes ([PR #1628](https://github.com/forcedotcom/salesforcedx-vscode/pull/1628))
+
+#### salesforcedx-vscode-core
+
+- Prevent `SFDX: Diff File Against Org` command from appearing when a non-SFDX project is opened ([PR #1608](https://github.com/forcedotcom/salesforcedx-vscode/pull/1608), [Issue #1600](https://github.com/forcedotcom/salesforcedx-vscode/issues/1600))
```
Comment (hbc-bindu) [inline packages/salesforcedx-vscode/CHANGELOG.md]: @brpowell , could we rephrase this as:
Fix `SFDX: Diff File Against Org` to appear in the Command Palette only when opening SFDX projects
Or it could be:
Prevent `SFDX: Diff File Against Org` from appearing in the Command Palette when opening non-SFDX projects.
